Hookies/Game/Network/NetworkManager.swift

```diff
@@ -142,22 +144,23 @@ class NetworkManager: NetworkManagerProtocol {
             guard let genericPlayerEventData = createPlayerEventData(from: genericPlayerAction) else {
                 return
             }
-
+            if genericPlayerEventData.playerData.playerId != currentPlayer?.playerId {
+                sendGenericPlayerEventNotification(genericPlayerEventData: genericPlayerEventData)
```

Comment:
I not very sure here ah. Cos I thought need to convert the playerEvent to the notification name, and then the userInfo in the notification needs to be a genericSystemEvent. So I refactored the sendGenericPlayerEventNotification(genericPlayerEventData: GenericPlayerEventData) to avoid duplication of code.